src/fieldset/fieldset.scss

```diff
@@ -14,7 +14,7 @@
 
   legend {
     font-size: $iui-font-size-leading;
-    padding: $iui-inputs-margin $iui-s;
+    padding: $iui-input-padding-height-small $iui-s;
```

Comment:
The fieldset legend was using a margin variable for padding.  I've replaced it with the correct variable.